modules/cifti_mesh_wf.nf

```diff
@@ -108,6 +108,8 @@ process ciftify{
     -v !{params.bids}:/bids \
     -v $(pwd):/output \
     -v $(pwd)/work:/work \
+    -v $(pwd)/fmriprep:/fmriprep \
+    -v $(pwd)/freesurfer:/freesurfer \
```

Comment:
```suggestion
```
Your invocation file is probably mis-specified here if you need to bind to the root directory of the container!

Reply:
Make sure that in ciftify that your fmriprep workdir is set to /output